### PR TITLE
Fix globalization tests to make it work on Windows 7

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
FYI @joshfree

File: src/System.Globalization.Extensions/tests/PlatformDetection.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+using System.Text;
+using Xunit;
+
+public static class PlatformDetection
+{
+    public static bool IsWindows { get; } = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+    public static bool IsWindows7 { get; } =  IsWindows && GetWindowsVersion() == 6 && GetWindowsMinorVersion() == 1;
+    public static int WindowsVersion { get; } = GetWindowsVersion();
+
+    private static int GetWindowsVersion()
```
Comment (joshfree) [inline src/System.Globalization.Extensions/tests/PlatformDetection.cs]: /cc @Priya91

**Comment by joshfree** [comment]
Thanks @tarekgh.  LGTM overall

**Comment by tarekgh** [comment]
test Innerloop OSX Release Build and Test please

**Comment by tarekgh** [comment]
test Innerloop OSX Release Build and Test please

**Comment by tarekgh** [comment]
test Innerloop Windows_NT Debug Build and Test  please

**Pull request merged**